docs/EXTENDS.md

```diff
@@ -0,0 +1,133 @@
+# Extending Configuration Files
+
+This module supports extending webpack configuration files with
+[ESLint-style](https://eslint.org/docs/user-guide/configuring#extending-configuration-files)
+`extends` functionality. A configuration file can extend the properties and
+collections from base configurations, and are filtered intelligently.
+
+The extends property value is either:
+
+- A `String` that specifies; a fully-qualified configuration path, or an NPM
+module name, or `require` path.
+- An `Array` of `String`. Each additional configuration extends the preceding
+configurations in the array. Configurations are extended recursively so a base
+configuration can also have an extends property.
+- An `Object` containing a `configs` property, which matches the two previously
+types, and a `filters` property, which contains an `Object` specifying filter
+rules.
+
+To demonstrate the concept, suppose we had the following simple setup:
+
+```js
+// base.config.js
+module.exports = {
+  name: 'base',
+  mode: 'development',
+  plugins: [...]
+}
+```
+
+```js
+// webpack.config.js
+module.exports = {
+  extends: path.join(..., 'base-config.js'),
+  name: 'dev'
+}
+```
+
+The resulting configuration object will look like:
+
+```js
+// result
+{
+  name: 'dev',
+  mode: 'development',
+  plugins: [...]
+}
+```
+
+Where `plugins` contains the plugins as defined in the `base.config.js` file.
+
+## Multiple Extended Configurations
+
+There might be situations where we just want to get crazy and extend multiple
+base or nested configurations. In that situation, doing so is as simple as
+specifying them in the order they should be applied. For example:
+
+```js
+// webpack.config.js
+module.exports = {
+  extends: [
+    path.join(..., 'batcave.config.js'),
+    path.join(..., 'wayne-tower.config.js'),
+    ...
+  ]
+};
+```
+
+## Using a Shareable Configuration Package
+
+A sharable configuration is an NPM package that exports a configuration object.
+Make sure the package has been installed to a directory where `config-loader`
+can `require` it.
+
+To use a sharable configuration, simply set the `extends` property equal to the
```

Comment:
Again, please remove _simply_

Reply:
what is the objection to use of that word?

Reply:
This is the one we're going to leave in. If someone is disheartened by setting a string value, they've got some work to do before tackling webpack. 😄